### PR TITLE
Makes things dust when the SM bumps them as well

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Makes things dust when a supermatter crystal runs into them as well. While this probably won't be the case with the main engine supermatter, it could come into play if a supermatter crystal starts moving around...
Also, provides some examine text to show that the crystal can be wrenched down, since it isn't really clear and not something you're generally going to experiment with.

## Why It's Good For The Game
Really, a supermatter crystal running into you and you being fine is just a weird interaction that doesn't feel quite right. This is more where it's at. Note that if an SM (in particular, a shard) runs into a window, it will destroy it, which should encourage you to anchor your shards before they start bouncing all over the place.

## Testing

https://github.com/ParadiseSS13/Paradise/assets/89928798/5bd0f82c-c44d-4454-ad6a-eb7e596e3334

## Changelog
:cl:
tweak: You will now be dusted if an SM crystal bumps itself into you. Watch out!
/:cl:

### CODE REVIEW

**Comment by DGamerL** [comment]
Very nice stuff!

**Comment by ArcticValor** [comment]
two words "SM cannon"

File: code/modules/power/engines/supermatter/supermatter.dm [hunk 1]
```diff
@@ -799,6 +801,10 @@
 	playsound(get_turf(src), 'sound/effects/supermatter.ogg', 50, TRUE)
 	Consume(AM)
 
+/obj/machinery/atmospherics/supermatter_crystal/Bump(atom/A, yes)
```
Comment (Henri215) [inline code/modules/power/engines/supermatter/supermatter.dm:804]: "yes"?

Comment (lewcc) [reply code/modules/power/engines/supermatter/supermatter.dm:804]: it's on the parent call like this as well /shrug

**Review by S34NW — APPROVED** [review]

**Pull request merged**